src/main/java/seedu/address/logic/commands/FreeCommand.java

```diff
@@ -39,7 +39,7 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
         requireNonNull(model);
 
         User user = model.getUser();
```

Comment:
Probably remove this line too then

Reply:
You're right, thanks!

Reply:
Correction, I think it's still being used in the line after `DeconflictTimeTable deconflict = new DeconflictTimeTable(user.getTimeTable());`

Reply:
ah, alright. missed that one